cmake/Packaging.cmake

```diff
@@ -139,6 +138,48 @@ ly_install(FILES ${_cmake_package_dest}
     COMPONENT ${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME}
 )
 
+# Do inital check on workspace path in the event that LY_3RDPARTY_PATH is misconfigured
+file(REAL_PATH "${CMAKE_CURRENT_SOURCE_DIR}/.." _root_path)
+if (EXISTS "${_root_path}/3rdParty/packages")
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY "${_root_path}/3rdParty/packages")
+else()
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY ${LY_3RDPARTY_PATH}/packages)
+endif()
+
+# Scan the engine and 3rd Party folders for licenses
+file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/python/python.cmd" _python_cmd)
+file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts/license_scanner" _license_script_path)
```

Comment:
We should use [cmake_path(SET ... NORMALIZE)](https://cmake.org/cmake/help/v3.21/command/cmake_path.html?highlight=cmake_path#cmake-path-set) instead file(TO_NATIVE_PATH) for path normalization in cmake now

Reply:
Good catch, I saw it was briefly mentioned [here](https://cmake.org/cmake/help/latest/command/file.html). I'll clean that up